src/sst/core/interfaces/simpleNetwork.h

```diff
@@ -174,7 +174,7 @@ class SimpleNetwork : public SubComponent
     {
 
     public:
-        SST_ELI_REGISTER_SUBCOMPONENT_API(SST::Interfaces::SimpleNetwork::NetworkInspector,std::string)
+        SST_ELI_REGISTER_SUBCOMPONENT_API(SST::Interfaces::SimpleNetwork::NetworkInspector, std::string)
```

Comment:
Looks like this one doesn’t follow convention and should be changed to one item per line at some point.